### PR TITLE
Draft blog post for v2.11 announcement

### PR DESCRIPTION
**Pull request by scotthmurray**
Please review and see if there are any other features or bug fixes you'd like to highlight.

### CODE REVIEW

File: site/blog/2023/v2.11.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## [v2.11](https://github.com/saulpw/visidata/releases/tag/v2.11) (2023-01-13)
+
+Hi, friends of VisiData!
+
+I’m happy to announce a new maintenance release 2.11. This will be the last release before some big, exciting changes introduced in 3.0.
+
+The most significant change is that 2.11 *drops support for Python 3.6* and *adds support for Python 3.11*.
+
+2.11 also includes several other small improvements and bug fixes, of which I’ll highlight two:
```
Comment (anjakefala) [inline site/blog/2023/v2.11.md:16]: This release is a bugfixing machine by Saul! I wish we had the count for how many issues we closed in the past 2-3 weeks. I think I want to emphasise that.

File: site/blog/2023/v2.11.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- `dup-sheet` now carries over attributes of columns added by `add-column` (helpful!)
+ - We added a new "concat" jointype. This behaves similar to "append", but keeps first sheet type and columns.
+
```
Comment (anjakefala) [inline site/blog/2023/v2.11.md:20]: Should we add that unless something comes up in the next 2 weeks or so, this is the version that we want to upload to Debian/Homebrew?

File: site/blog/2023/v2.11.md (hunk 1 above)
Comment (scotthmurray) [reply site/blog/2023/v2.11.md:16]: Are there specific bugs in the changelog that would be good to highlight? (Anything that's been bothering some users?)

File: site/blog/2023/v2.11.md (hunk 2 above)
Comment (scotthmurray) [reply site/blog/2023/v2.11.md:20]: Not my call, but that sounds like a great idea to me. Very good to push 2.11 (as the last release before the big 3.0) to Debian/Homebrew and make updating easier for those users.

**Review by saulpw — APPROVED** [review]
Thanks so much for putting this together, @scotthmurray !

**Comment by scotthmurray** [comment]
Super, glad this worked. :)

**Pull request merged**